### ISSUE TITLE
built-in dbt config macro doesn't support config.get()

### ISSUE THREAD

**Issue by sethwoodworth** [sqlfluff/sqlfluff#516]
The dbt `config` macro allows you to use the config object in your model and provides a `get` function like this: `{{ config.get('materialization', default='incremental') }}`.  If I use `config.get` in a dbt model and run sqlfluff I get the error:

```
== [filename.sqll] FAIL
L:   - | P:   - |  TMP | Unrecoverable failure in Jinja templating:
                       | 'jinja2.runtime.Macro object' has no attribute 'get'.
                       | Have you configured your variables?
                       | https://docs.sqlfluff.com/en/latest/configuration.html
```

**Comment by sethwoodworth** [comment]
Like many of the issues with sqlfluff and dbt, this is very likely solved by #371 & #508.

**Comment by NiallRees** [comment]
Resolved by #508 to be fixed in 0.4.0
